tests/common/include/tests/common/energy_loss.inl

```diff
@@ -79,10 +81,13 @@ INSTANTIATE_TEST_SUITE_P(
     ::testing::Values(std::make_tuple(helium_gas<scalar>(),
                                       0.1003f * unit<scalar>::GeV, 3.082f)));
 
+/*
+//@ NOTE: Test fails with He Gas and 10 GeV muons (18 % difference)
```

Comment:
It is where the test fails actually. btw, failing with He gas is not very fun :/